gems/pending/util/duplicate_blocker/dedup_handler.rb

```diff
@@ -63,8 +68,8 @@ def handle(method, *args)
       key = key_generator.call(method, *args)
       desc = descriptor.call(method, *args)
 
-      entry = update_history(key, desc)
-      tripped?(entry) ? on_blocking_call(desc) : method[*args]
+      entry = update_history(key.hash, desc)
```

Comment:
I think it would make more sense if the default key_generator generated the hash, if that's what you're looking for.  On that note, do you actually use the ability to use an alternate key generator? (line 87)